### DISCUSSION TITLE
get files from channel

### DISCUSSION THREAD

**Discussion by yousefjf** [Nekmo/telegram-upload#155] [Q&A]
How can I set a channel for getting files instead of saved messages?

**Comment by tissole** [comment]
`telegram-download -f channel_name`   Replace `channel_name `with the actual name of the channel you want to download from. This downloads all the files and works for public channels. Another option is `telegram-download -i` to download interactively. You have to choose the channel from the menu and the files you want.

**Comment by ArianKhadem** [comment]
is there any way to download from private channels too?

**Reply by qbht** [reply]
you can download private channel if you are owner  telegram-download -f https://t.me/+private2332channel

**Reply by prisonerr627** [reply]
> you can download private channel if you are owner  telegram-download -f https://t.me/+private2332channel

How did you got the link I have tried to give the link but it didn't worked. Well I'm not the owner of the channel but I have access to the channel to download. Please check my issue for screenshots --> https://github.com/Nekmo/telegram-upload/issues/247#issue-2073128113
